mParticle-Kochava/MPKitKochava.m

```diff
@@ -2,11 +2,17 @@
 #import "MPKochavaSpatialCoordinate.h"
 #import "mParticle.h"
 #import "MPKitRegister.h"
+#import "MPIConstants.h"
 #if defined(__has_include) && __has_include(<KochavaTracker/KochavaTracker.h>)
 #import <KochavaTracker/KochavaTracker.h>
 #else
 #import "KochavaTracker.h"
 #endif
+#if defined(__has_include) && __has_include(<KochavaAdNetwork/KochavaAdNetwork.h>)
+#import <KochavaAdNetwork/KochavaAdNetwork.h>
```

Comment:
This change is also breaking SPM--maybe this needs to be one single chain using e.g. `elif`?

Reply:
Seems odd that it would be fine on line 6 but not here. Mind checking SPM again, it doesn't seem to occur in my setup.